scripts/prepdocs.sh

```diff
@@ -12,7 +12,7 @@ $(azd env get-values)
 EOF
 
 echo 'Creating python virtual environment "scripts/.venv"'
-python -m venv scripts/.venv
+python3 -m venv scripts/.venv
```

Comment:
Would you mind doing the same for: https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/app/start.sh#L20 ?

Reply:
I think this is a good improvement.

In the future, we might want to update the sh script to do auto-detect..  Similar to what we do for the pwsh version (https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/scripts/prepdocs.ps1#L19)

Reply:
Sure, updated start.sh. I'm curious what situations there are where python is preferred over python3, as we currently have in our internal Python team style guide to prefer the explicit python3 command. I suppose someone might map python to a particular install of python3, and thats the situation you're trying to support?

Reply:
PEP394 is the best description of the aliases that Linux distributions follow https://peps.python.org/pep-0394/